source/MaterialXGenShader/GenOptions.h

```diff
@@ -75,7 +75,7 @@ class GenOptions
 
     /// Sets the method to use for specular environment 
     /// lighting for HW shader targets.
-    unsigned int hwSpecularEnvironmentMethod;
+    int hwSpecularEnvironmentMethod;
 
```

Comment:
Decided to go with int's for the option enums because supporting values other than defined enum values is useful here. Derived generators can then implement new custom interface types and environment methods, and list them as new values on the options.